assets/js/translator.js

```diff
@@ -6,6 +6,7 @@ var curSrcLang /*: string */, curDstLang/*: string */;
 var recentSrcLangs /*: string[] */ = [], recentDstLangs /*: string[] */ = [];
 var droppedFile/*: ?File */;
 var translateRequest;
+var timer;
```

Comment:
Could you call it something a bit more specific like `translationTimer` since it's a global? Also, I think it can be typed.